app/uk/gov/hmrc/gform/testonly/TestOnlyController.scala

```diff
@@ -83,4 +90,32 @@ class TestOnlyController(mongo: () => DB, enrolmentConnector: EnrolmentConnector
     enrolmentConnector.removeUnallocated(user.id).map(_ => NoContent)
   }
 
+  def testValidatorStub(utr: String) = Action.async { implicit request =>
```

Comment:
TestValidatorStub and TestGetTaxPeriods stub should not be needed any more.  gform-stubs is now doing this